pkg/manage/cmd_setup.go

```diff
@@ -0,0 +1,108 @@
+package manage
+
+import (
+	"context"
+	"crypto/rand"
+	"encoding/base64"
+	"fmt"
+	"io/fs"
+	"os"
+	"path"
+
+	"github.com/adrg/xdg"
+	"github.com/spf13/cobra"
+)
+
+const setupHelp = `Builds required files and docker images
+
+This command executes the following steps to start OpenSlides:
+- Create a local docker-compose.yml.
+- Create local secrets for the auth service.
+- Creates the services.env.
+- Runs docker-compose build to build images. TODO
+`
+
+const appName = "openslides"
+
+// CmdSetup creates docker-compose.yml, secrets and services.env. Also runs
+// docker-compose build to build all images.
+func CmdSetup(cfg *ClientConfig) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "setup",
+		Short: "Builds the required files and docker images.",
+		Long:  setupHelp,
+	}
+
+	local := cmd.Flags().BoolP("local", "l", false, "Create required files in currend working directory.")
+
+	cmd.RunE = func(cmd *cobra.Command, args []string) error {
+		ctx, cancel := context.WithTimeout(context.Background(), cfg.Timeout)
+		defer cancel()
+
+		dataPath := path.Join(xdg.DataHome, appName)
+		if *local {
+			p, err := os.Getwd()
+			if err != nil {
+				return fmt.Errorf("getting current directory: %w", err)
+			}
+			dataPath = p
+		}
+
+		if err := os.MkdirAll(dataPath, fs.ModePerm); err != nil {
+			return fmt.Errorf("creating directory `%s`: %w", dataPath, err)
+		}
+
+		if err := createDockerComposeYML(ctx, dataPath); err != nil {
+			return fmt.Errorf("creating Docker Compose YML: %w", err)
+		}
+
+		if err := createEnvFile(dataPath); err != nil {
```

Comment:
Test if .env works

Reply:
At the moment I think it would be more explicit if we let `services.env` so the minor expert user can see this file and find it referenced in docker-compose.yml